src/xdocs/config_coding.xml

```diff
@@ -3614,6 +3614,99 @@ class TestHashCode {
       </subsection>
     </section>
 
+    <section name="MatchXpath">
+      <p>Since Checkstyle 8.37</p>
+      <subsection name="Description" id="MatchXpath_Description">
+        <p>
+          Evaluates Xpath query and report violation on all matching AST nodes. Current check allows
+          user to implement custom checks using Xpath.
+          Please read more about xpath syntax at
+          <a href="https://www.w3schools.com/xml/xpath_syntax.asp"> W3Schools Xpath Syntax</a>.
+        </p>
+      </subsection>
+
+      <subsection name="Properties" id="MatchXpath_Properties">
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>query</td>
+              <td>Xpath query.</td>
+              <td><a href="property_types.html#String">String</a></td>
+              <td><code>&quot;&quot;</code></td>
+              <td>8.37</td>
+            </tr>
+          </table>
+        </div>
+      </subsection>
+
+      <subsection name="Examples" id="MatchXpath_Examples">
+        <p>
+          To configure the check with default configuration:
```

Comment:
Do we need default configuration here? It does nothing. Similar checks has [no default configuration](https://checkstyle.org/config_coding.html#IllegalTokenText_Examples)

Reply:
done